code/__DEFINES/is_helpers.dm

```diff
@@ -172,7 +172,7 @@ GLOBAL_LIST_INIT(turfs_without_ground, typecacheof(list(
 
 #define ismecha(A) (istype(A, /obj/mecha))
 
-#define ismopable(A) (A.layer <= HIGH_SIGIL_LAYER) //If something can be cleaned by floor-cleaning devices such as mops or clean bots
+#define ismopable(A) (A?.layer <= HIGH_SIGIL_LAYER ? TRUE : FALSE) //If something can be cleaned by floor-cleaning devices such as mops or clean bots
```

Comment:
null's behavior in math operations is undefined.

```suggestion
#define ismopable(A) (A && (A.layer <= HIGH_SIGIL_LAYER)) //If something can be cleaned by floor-cleaning devices such as mops or clean bots
```
This is much cleaner.

Reply:
This is better, thank you MSO !